portfolio/src/main/webapp/script-home.js

```diff
@@ -33,38 +33,69 @@ window.onload = function() { main(); }
+  const /** ?HTMLCollection */ slideshowBlog  =
+      new SlideShow(document.getElementsByClassName('blog-slides'));
+  slideshowBlog.setToAutomaticallyChangeSlides();
 
-    document.getElementById('switch-slides-left').onclick =
+  document.getElementById('switch-slides-left').onclick =
         function adjustBackOne() {
           slideShowGallery.adjustSlideManual(ADJUST_BACK); 
-    }
-    document.getElementById('switch-slides-right').onclick =
+        };
+  document.getElementById('switch-slides-right').onclick =
         function adjustForwardOne() {
           slideShowGallery.adjustSlideManual(ADJUST_FORWARD); 
-    }
-    // Fetches the pre-made comment list from the servlet and makes 
-    // a list for each comment.
-    fetch('/comments').then(response => response.json()).then((comments) => {
-      const /** ?HTMLCollection */commentContainer =
-            document.getElementById('comments-container');
-      comments.forEach(function(comment) {
-          commentContainer.appendChild(createListElement(comment));
-      });
-    });
+        };
 }
 
-/*
- * @return the <li> element containing text passed in.
- * @param {string} text is what is put into the <li>
+/** 
+ * function populateComments() populates the comment board on the webpage.
  */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+function populateComments() {
+  fetch('/comments').then(response => response.json()).then(
+        (comments) => {
+          const /** ?HTMLCollection */commentContainer =
+              document.getElementById('comments-container');
+
+          comments.forEach(function(comment) {
+            let /** string */ stringOfName;
+            if (comment.name==null || comment.name=='') {
+              stringOfName = 'Anonymous';
+            } else {
+              stringOfName = comment.name;
+            }
+            const /** ?HTMLCollection */ nameOfCommenter =
+                document.createElement('h3');
+            nameOfCommenter.innerHTML = stringOfName;
+            const /** ?HTMLCollection */ dateOfComment =
+                document.createElement('h4');
+            dateOfComment.innerHTML =
+                "Date Posted: " + comment.timeOfComment;
+            const /** ?HTMLCollection */ commentString =
+                document.createElement('p');
+            commentString.innerHTML = comment.commentString;
+
+            //Adds the individual elements to a single div
+            const /** ?HTMLCollection */ divOfComment =
+                document.createElement('div');
+            divOfComment.appendChild(nameOfCommenter);
+            divOfComment.appendChild(dateOfComment);
+            divOfComment.appendChild(commentString);
+
+            divOfComment.style.border='3px solid #b31b1b';
+            divOfComment.style.margin='15px 0 15px';
+            divOfComment.style.padding='10px';
```

Comment:
Everything involving the creation of the div for a comment should be in another function for readability. So like, everything from line 76 to 96. The function should return the div element and take in as a parameter, the parts of the comment